Client.php

```diff
@@ -1226,9 +1226,26 @@ public function __call($name, $args)
                     throw $e;
                 }
             } else {
-                $this->write_command($command);
-                $response = $this->read_reply($name);
-                $response = $this->decode_reply($name, $response, $trackedArgs);
+                $retryCount = 0;
+                $maxRetries = 5;
```

Comment:
A hard-coded retry count really isn't appropriate, this should be linked to a configurable and probably with a lower limit or disabled by default.